client/src/pages/Profile.js

```diff
@@ -9,14 +9,39 @@ import { withPageContext } from '../components/pageContext';
 
 import './profile.css';
 
+const styles = {
+  pageContent: {
+    display: 'flex',
+  },
+  headerContainer: {
+    display: 'flex',
+    flexDirection: 'row',
+  },
+  header: {
+    display: 'inline',
+    margin: '10px',
+    padding: '20px',
+    '&:hover': {
+      cursor: 'pointer',
+      color: '#E4E8EA',
```

Comment:
I just looked at the design on local and think it fits much better to change the color to our primary color you can do that by wrapping styles in a function ex:
(you might need to do ({theme}) instead of (theme) I forget...
```
const styles = (theme) => ({...})
```

and then for the color do something like `color: theme.palette.primary.main`

Reply:
Just changed it